package.json

```diff
@@ -13,14 +13,14 @@
     "LICENSE"
   ],
   "scripts": {
-    "lint": "# tslint \"src/**/*.ts\"",
-    "build": "rm -rf dist/ && tsc",
+    "lint": "echo deactivated for now: tslint src/**/*.ts",
+    "build": "rimraf dist && tsc",
     "dependency-check": "dependency-check . --entry dist/bin/typings-bundle.js --entry dist/bin/typings-init.js --entry dist/bin/typings-install.js --entry dist/bin/typings-list.js --entry dist/bin/typings-search.js --entry dist/bin/typings-uninstall.js --unused --no-dev -i bluebird",
     "test-spec": "ts-node node_modules/blue-tape/bin/blue-tape.js \"src/**/*.spec.ts\" | tap-spec",
     "test-cov": "ts-node node_modules/istanbul/lib/cli.js cover -e .ts --print none -x \"*.d.ts\" -x \"*.spec.ts\" blue-tape -- \"src/**/*.spec.ts\" | tap-spec",
     "test": "npm run lint && npm run build && npm run dependency-check && npm run test-cov",
-    "bootstrap": "mkdir -p typings && touch typings/main.d.ts && SKIP_PREPUBLISH=true npm install && npm run build 2> /dev/null; cat dist/init.js > /dev/null && node dist/bin/typings.js install",
-    "prepublish": "if [ ! $SKIP_PREPUBLISH ]; then npm run build; fi"
+    "bootstrap": "npm install shelljs && node scripts-bootstrap.js && node dist/bin/typings.js install",
```

Comment:
But wouldn't not installing TypeScript now rely on the global version instead of the local version? That's my only concern, need to make sure boostrapping is using the right dependencies.